liquidator/liquidator.js

```diff
@@ -51,17 +52,19 @@ class Liquidator {
         address: position.sponsor
       });
 
-      // Create the liquidation transaction
-
-      // TODO calculate the amountToLiquidate as a function of the total collateral within the position
-      // and the current price of the collateral. This will require knowing how much the collateral and the
-      // synthetic are worth.
+      // Create the liquidation transaction to liquidate the entire position:
+      // - Maximum (Collateral/Synthetic) ratio to liquidate.
+      let liquidateCollateralPerToken = (parseFloat(position.amountCollateral) / parseFloat(position.numTokens));
+      // - Maximum amount of Synthetic tokens to liquidate.
+      let liquidateTokens = position.numTokens;
```

Comment:
good stuff. I had not filled out this function because prasad was mid partial liquidations when I got this PR in. this looks perfect 👍